.github/workflows/BlackFormat.yml

```diff
@@ -12,8 +12,37 @@ jobs:
     name: Black Formatter for Python Exporter
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: psf/black@stable
+      - name: Checkout Code
+        uses: actions/checkout@v2
+      - name: Python Setup
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+      - name: Setup Isort
+        run: python3 -m pip install isort
+      - name: Validate Isort Formatting
+        run: |
+          python - <<'EOF'
+          import os, subprocess, sys
+
+          EXPORTER_DIR = "exporter/SynthesisFusionAddin"
+          files = (lambda d: [os.path.abspath(os.path.join(root, filename)) for root, _, filenames in os.walk(d)
+                   for filename in filenames if filename.endswith(".py")])(EXPORTER_DIR)
+          oldFileStates = [open(file, "r").readlines() for file in files]
+          subprocess.call(["isort", EXPORTER_DIR], bufsize=1, shell=False)
+          newFileStates = [open(file, "r").readlines() for file in files]
+          exitCode = 0
+          for i, (oldFileState, newFileState) in enumerate(zip(oldFileStates, newFileStates)):
+            for j, (previousLine, newLine) in enumerate(zip(oldFileState, newFileState)):
+              if previousLine != newLine:
+                print(f"File {files[i]} is not formatted correctly!\nLine: {j + 1}")
+                exitCode = 1
+
+          if not exitCode: print("All files are formatted correctly with isort!")
+          sys.exit(exitCode)
+          EOF
```

Comment:
Could we just make this a `.py` file and call it with the workflow?

Reply:
@HunterBarclay

Yes we can, would you prefer that? I would have put it in the `scripts/` folder but that was being removed so I wanted to avoid conflicts.

Reply:
@BrandonPacewic I would definitely put it in the exporter directory. Maybe you want to add that plus a script to run black and isort?

Reply:
@HunterBarclay How would you like the formatting scripts done? `.bat` and `.sh` file somewhere? A python file? Build it into the main `Synthesis.py` file?

Reply:
I think python is fine for this instance.

Reply:
Python file it is.